packages/csv-parser-price/src/main.js

```diff
@@ -61,8 +61,25 @@ export default class CsvParserPrice {
           strict: true,
         })
       )
+      .map(a => {
+        if (!a[CONSTANTS.header.sku]) {
```

Comment:
Cool, I didn't know that SKU is mandatory when working with these tools.

Reply:
@daern91 How can we be sure about it? Was this fix requested by the only customer using this importer? I think that this change is at least a breaking change and have doubts that it works for every existing customer. What do you think?

Reply:
@danie1sullivan  @katmatt What I understood from the code is we push price corresponding to the sku provided. And even there are code below dependent on sku-id. So my assumption was if there is not sku provided then we won't be able to update anything.
This my understanding, I may be wrong please correct me if I am.

Reply:
Hi again @katmatt and @VineetKumarKushwaha,

Yes below these lines I also found that we use [highland.reduce](https://highlandjs.org/#reduce) which takes an initial source value and an iterator. 

https://github.com/commercetools/nodejs/blob/1292a8c3f248a0815cbb9e8e3e614aef4a408213/packages/csv-parser-price/src/main.js#L83-L111

The iterator sent in is `this.mergeBySku`
https://github.com/commercetools/nodejs/blob/1292a8c3f248a0815cbb9e8e3e614aef4a408213/packages/csv-parser-price/src/main.js#L182-L201
It seems like we would throw above if an SKU was not present. So I guess this tool only works with SKUs and couldnt add prices with variant ID numbers.

The [example in the docs](https://commercetools.github.io/nodejs/cli/csv-parser-price.html#configuration) also use SKU

Reply:
@daern91 Thanks for your explanation and then I'm fine with the changes 👍